features/integration-github-bitbucket-gitlab.md

```diff
@@ -41,7 +41,7 @@ Private Packagist integrates with the following systems:
 * Webhooks: Code changes and releases
 
 #### Bitbucket Data Center / Server
-* OAuth: Users authenticate on Private Packagist with their Bitbucket Data Center / Server accounts. Please contact us to set this up for the cloud plan or follow [this setup](../docs/self-hosted/bitbucket-server-integration-setup.md) for Self-Hosted.
+* OAuth: Users authenticate on Private Packagist with their Bitbucket Data Center / Server accounts. Follow [this setup](../docs/cloud/bitbucket-server-integration-setup.md) for the cloud plan or [this setup](../docs/self-hosted/bitbucket-server-integration-setup.md) for Self-Hosted.
```

Comment:
```suggestion
* OAuth: Users authenticate on Private Packagist with their Bitbucket Data Center / Server accounts. Follow [this setup](../docs/cloud/bitbucket-server-integration-setup.md) for Cloud plans or [this setup](../docs/self-hosted/bitbucket-server-integration-setup.md) for Self-Hosted.
```

Reply:
oh! you suggested for all places. Didn't see. Updated via commit and took the suggestion literally.